app/views/audits/types/_additional_document_validation_request_received.html.erb

```diff
@@ -1,5 +1,4 @@
 <%= activity(locals[:item].activity_type, locals[:item].activity_information) %>
-<p class="govuk-body">
```

Comment:
this needs the p-govuk-body tag otherwise it shows without formatting in the audit log accordion, can you please re-add it?
![image](https://user-images.githubusercontent.com/9452321/146179329-04ed696e-2114-4bcb-8e21-61bf20e12524.png)

Reply:
woops total accident. Nice catch!